docs/index.md

```diff
@@ -0,0 +1,39 @@
+# Calliope: energy system modelling made simple
+
+!!! note
+
+    This is the documentation for version {{ version }} ([Version history](version_history.md)). See the [main project website at www.callio.pe](https://www.callio.pe/) for more general information, including a gallery of models built with Calliope, and other useful information.
+
+Calliope focuses on flexibility, high spatial and temporal resolution, the ability to execute many runs based on the same base model, and a clear separation of framework (code) and model (data). Its primary focus is on planning energy systems at scales ranging from urban districts to entire continents. In an optional operational mode it can also test a pre-defined system under different operational conditions. Calliope's built-in tools allow interactive exploration of results.
```

Comment:
It would be good to follow the convention of one sentence per line so that future git diffs are simpler. [Here's a proponent](https://nick.groenen.me/notes/one-sentence-per-line/)

Reply:
👍